packages/node/src/eventbuilder.ts

```diff
@@ -0,0 +1,106 @@
+import { getCurrentHub } from '@sentry/core';
+import { Event, EventHint, Mechanism, Severity } from '@sentry/types';
+import {
+  addExceptionMechanism,
+  addExceptionTypeValue,
+  extractExceptionKeysForMessage,
+  isError,
+  isPlainObject,
+  normalizeToSize,
+  SyncPromise,
+} from '@sentry/utils';
+
+import { extractStackFromError, parseError, parseStack, prepareFramesForEvent, ReadFileFn } from './parsers';
+import { NodeOptions } from './types';
+
+/**
+ * Builds and Event from a Exception
+ * @hidden
+ */
+export function eventFromException(
+  options: NodeOptions,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+  exception: any,
+  readFile?: ReadFileFn,

```

Comment:
I'd prefer to keep function signature as close to the one from browser as possible, so I'd swap the order of `readFile` and `hint`.